### ISSUE TITLE
amc.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#3554]
### Basic Information

**Test link:** `https://www.amc.com/`
**Category:** anti-adblock
**Reported from:** United Arab Emirates
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:75.0) Gecko/20100101 Firefox/75.0
**Extension:** Nano Defender for Firefox 15.0.0.197

### Additional Message

Try watching anything and you would get the anti-adblock message.

**Comment by mapx-** [comment]
> This episode is not available due to licensing restrictions. We know, it’s a bummer. To find out where you can watch it, visit amc.tv/BCSHelp

**Comment by llacb47** [comment]
Can't reproduce, there is unblocked sourcepoint domain `mms.amc.com`.

**Comment by mapx-** [comment]
yeah, other series can be seen without issues (using a US IP), for example
`https://www.amc.com/shows/dispatches-from-elsewhere/full-episodes/season-01/episode-07/cave-of-kelpius`

**Comment by jspenguin2017** [comment]
Works fine for me. 
![image](https://user-images.githubusercontent.com/7283682/79129110-dc53ca80-7d61-11ea-98de-9d2f4aae585d.png)

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.amc.com/shows/better-call-saul/full-episodes/season-05/episode-10/something-unforgivable`
**Category:** anti-adblock
**Reported from:** United Arab Emirates
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:75.0) Gecko/20100101 Firefox/75.0
**Extension:** Nano Defender for Firefox 15.0.0.197

### Additional Message

**Comment by jspenguin2017** [comment]
Still works fine for me.
